.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/215865888-ec25f3de-d826-4861-80b4-d6e67ee5e1c3.png)

Reply:
I thought that there is no need to leave)